api_compatibility_policy.md

```diff
@@ -73,10 +61,8 @@ incompatible change is made.
 
 There are two reasons for this:
 
-- `Task` and `TaskRun` are considered upgraded versions of `Build`, meaning that
-  the APIs benefit from a significant amount of user feedback and iteration
-- Going forward users should use `TaskRun` and `Task` instead of `Build` and
-  `BuildTemplate`, those users should not expect the API to be changed on them
+- `Task` and `TaskRun` are considered upgraded versions of [Build](https://github.com/knative/docs/blob/master/docs/build/builds.md#source) and [BuildTemplate](https://github.com/knative/docs/blob/master/docs/build/build-templates.md), meaning that the APIs benefit from a significant amount of user feedback and iteration
```

Comment:
When we refer to the link in the docs, we are recommended to use the link under knative.dev/docs.
For example, Build should refer to https://knative.dev/docs/build/builds/(I am not sure we should use https://knative.dev/docs/build/builds/#source), and BuildTemplate goes to https://knative.dev/docs/build/build-templates/.

Reply:
Minor nit, I think it would be good to qualify build and buildtemplate as "Knative Build and BuildTemplate"